### PR TITLE
Add EEA token execution logic application with updated license

### PR DESCRIPTION
**Pull request by reckeyzhang**
Signed-off-by: reckeyzhang <lei.zhang@iex.ec>

### CODE REVIEW

**Comment by danintel** [comment]
Next time instead of opening a new PR, force push (`push -f`) the current PR if you need to add a DCO line to your commit.

File: examples/apps/eea_token/readme [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
+EEA token execution logic allows running the execution logic based on token requests,
+alse see check https://github.com/EntEthAlliance/Trusted-Token token_validation.cpp
```
Comment (danintel) [inline examples/apps/eea_token/readme]: Copied from the old PR #52:

> manojgop yesterday  Member
> The link https://github.com/EntEthAlliance/Trusted-Token is broken. Is there any link where we can find more details about trusted token and about functionality implemented in this PR
> 
>   @danintel
> danintel yesterday  Member
> Sanjay Bakshi may have a link.
> I do not know of any.
> 
>   @reckeyzhang
> reckeyzhang yesterday  Author Member
> This link is right and the github project is currently private
> 
>   @danintel
> danintel yesterday  Member
> I confirmed the link is correct with Sanjay.

File: examples/apps/eea_token/execution_logic.h [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#define ERROR_SHARE_TOKENS                          4
+
+#define ACTIVITY_PARTICIPATION_MEETING              1
+#define ACTIVITY_CHAIR_MEETING                      2
+#define ACTIVITY_CONTRIBUTE_DELIVERABLE             3
+#define ACTIVITY_EDITOR_DELIVERABLE                 4
+#define ACTIVITY_CONTRIBUTE_EEA_PROJECT_REGULAR     5
+#define ACTIVITY_CONTRIBUTE_EEA_PROJECT_MANAGER     6
+#define ACTIVITY_CONTRIBUTE_EEA_PROJECT_SPONSOR     7
+#define ACTIVITY_NUMBER                             7
```
Comment (danintel) [inline examples/apps/eea_token/execution_logic.h]: Copied from PR #38 

> danintel 8 days ago  Member
> ACTIVITY_NUMBER is 7, the same as ACTIVITY_CONTRIBUTE_EEA_PROJECT_SPONSOR. I suggest changing it to 8.
> 
>   @reckeyzhang
> reckeyzhang 7 days ago  Author Member
> Actually, ACTIVITY_NUMBER represents the current total number of activities (1-7), we do not name the activity type starting with 0 in order to conform with the activity type indices defined by EEA token spec.

**Comment by danintel** [comment]
Previously filed as PR #38 and PR #52.

**Comment by manojgop** [comment]
Is there any  read me to understand about EEA token ?

**Pull request merged**